crates/next-core/src/next_font/google/options.rs

```diff
@@ -99,8 +99,9 @@ pub(super) fn options_from_request(
         })
         .unwrap_or_default();
 
+    let supports_variable_weight = font_data.weights.iter().any(|el| el == "variable");
```

Comment:
I hoisted this (as it's used in two places) and switched this to `.iter().any(|el| el == "variable")` from `.contains(&"variable".into())` to avoid the unnecessary conversion to `RcStr` from `.into()`.

`HashSet::contains` accepts `&Q` where `T: Borrow<Q>`, which would allow it to take `&str` in this case instead of `&RcStr`, but sadly, `slice::contains` does no such thing.